server/routes/api/babysitter/babysitterRoutes.ts

```diff
@@ -38,4 +52,129 @@ babysitterRouter.get(
+      if (!validation.isValid) {
+        return res.status(400).json({ error: validation.message });
+      }
+
+      const imageName = `babysitter_${babysitterId}`;
+
+      if (!req.file) {
+        return res.status(400).json({ error: `Missing file input.` });
+      }
+      
+      const response = await s3.putImage(req.file, imageName);
+      if (!response) {
+        return res.status(500).json({ message: `Couldn't upload image` });
+      }
+      await handler.putProfileImage(imageName, Number(babysitterId));
+
+      return res.status(200).json({ message: `Image profile uploaded.` });
+    } catch (e) {
+      console.log(
+        `Error message: ${req.body.id}: ${(e as Error).message}\n${
+          (e as Error).stack
+        }`
+      );
+      return res.status(500).end();
+    }
+  }
+);
+
+babysitterRouter.get(
+  "/image/:babysitterId",
+  [
+    param("babysitterId").notEmpty().isNumeric().withMessage(BABYSITTER_INVALID_INPUT_ERROR)
+  ],
+  async (req: Request, res: Response) => {
+    try {
+      const fieldValidationResult = validationResult(req);
+      if (!fieldValidationResult.isEmpty()) {
+        return res.status(400).json({
+          message: fieldValidationResult
+            .array()
+            .map((item) => item.msg)
+            .join(" "),
+        });
+      }
+      const { babysitterId } = req.params;
+      const validation = await handler.userValidation(Number(babysitterId));
+      if (!validation.isValid) {
+        return res.status(400).json({ error: validation.message });
+      }
+
+      const imageName = await handler.getProfileImage(Number(babysitterId));
+      if (!imageName) {
+        return res.status(204).json({ message: `User doen't have an image` });
+      }
+      
+      const localStackImageUrl = await s3.getImageUrl(imageName);
+      const imageUrl = localStackImageUrl.replace('s3-local', s3.s3Uri);
+
+      res.status(200).json({ imageUrl });
+    } catch (e) {
+      console.log(
+        `Error message: ${req.body.id}: ${(e as Error).message}\n${
+          (e as Error).stack
+        }`
+      );
+      return res.status(500).end();
+    }
+  }
+);
+babysitterRouter.delete(
+  "/image/:babysitterId",
+  [
+    param("babysitterId").notEmpty().isNumeric().withMessage(BABYSITTER_INVALID_INPUT_ERROR)
+  ],
+  async (req: Request, res: Response) => {
+    try {
+      const fieldValidationResult = validationResult(req);
+      if (!fieldValidationResult.isEmpty()) {
+        return res.status(400).json({
+          message: fieldValidationResult
+            .array()
+            .map((item) => item.msg)
+            .join(" "),
+        });
+      }
+      const { babysitterId } = req.params;
+      const validation = await handler.userValidation(Number(babysitterId));
+      if (!validation.isValid) {
+        return res.status(400).json({ error: validation.message });
+      }
+      const imageName = await handler.getProfileImage(Number(babysitterId));
```

Comment:
all three of these commands i would also run in an handler function called deleteProfileImage - that runs thee 3 commands